Example/EpoxyExample/ViewControllers/TextFieldViewController.swift

```diff
@@ -0,0 +1,52 @@
+// Created by oleksandr_zarochintsev on 4/26/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import Epoxy
+import UIKit
+
+final class TextFieldViewController: CollectionViewController {
+
+  // MARK: Lifecycle
+
+  init() {
+    super.init(layout: UICollectionViewCompositionalLayout.listNoDividers)
+    setSections([.init(items: [textFieldRowItem])], animated: true)
+  }
+
+  // MARK: Internal
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    bottomBarInstaller.install()
+  }
+
+  // MARK: Private
+
+  private enum DataID {
+    case username
+    case button
+  }
+
+  private lazy var bottomBarInstaller = BottomBarInstaller(
+    viewController: self,
+    avoidsKeyboard: true,
+    bars: [buttonRowBar])
+
+  private var textFieldRowItem: ItemModeling {
+    TextFieldRow.itemModel(
+      dataID: DataID.username,
+      content: .init(
+        placeholder: "Username"),
+      style: .base)
+  }
+
+  private var buttonRowBar: BarModeling {
+    ButtonRow.barModel(
+      dataID: DataID.button,
+      content: .init(text: "Buy now"),
```

Comment:
nit: maybe "submit" to better match the "username" field? I could imagine this demo would make sense as a fake login page if you'd want to add a fake password field too
```suggestion
      content: .init(text: "Submit"),
```